documentation/internal/Overview-of-Navigation-&-URL-structure.md

```diff
@@ -65,15 +63,15 @@ Example: [https://balsamiq.cloud/scs7giw/ponj59g/r4C62](https://balsamiq.cloud/s
 
 ---
 
-## `/applications/new`
+## `/application/new`
```

Comment:
I'm confused, in the PR description it says `applications/new`. Anyway when I try `application/new` the page is not found. :(

Reply:
Oh. My bad. I hasn't changed the branch correctly.